.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: charmed-hpc-terraform tests
+on:
+  workflow_call:
+  pull_request:
+
+jobs:
+  inclusive-naming-check:
+    name: Inclusive naming check
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Run tests
+        uses: get-woke/woke-action@v0
+        with:
+          fail-on-error: true
+
+  validation-test:
+    name: Validation tests
+    runs-on: ubuntu-24.04
+    needs:
+      - inclusive-naming-check
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          sudo snap install --classic opentofu
+          sudo apt-get update
+          sudo apt-get install -y just
+      - name: Run tests
+        env:
+          TERM: xterm-256color
```

Comment:
[nit]: Do we need this environment variable declared because `just` outputs color codes that are incompatible with GitHub Actions?

Reply:
It's for the bold formatting we have in `justfile`. Line `` bold := `tput bold` `` fails with `tput: No value for $TERM and no -T specified` otherwise. Specifically `xterm-256color` was chosen as it's the default on my 24.04 machine.

Reply:
Hmm... seems like we should possibly consider removing the syntactic sugar. I thought the bold was cool :sunglasses::tm:, but perhaps we should remove it, especially if we need to declare the `TERM` env var in every CI job we have that uses a `justfile` at some point.

We do need to the quick `justfile` spec as suggested by @jedel1043, but that'll likely be a later in the cycle thing as we do QA testing for Charmed HPC.